### PR TITLE
max_client_connections ->maxClientCnxns

### PR DESCRIPTION
**Pull request by emalloy**
introduce ansible default var max_client_connections to inject maxClientCnxns into zoo.cfg template

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ tick_time: 2000
 zookeeper_autopurge_purgeInterval: 0
 zookeeper_autopurge_snapRetainCount: 10
 zookeeper_cluster_ports: "2888:3888"
+max_client_connections: 60
```
Comment (ernestas-poskus) [inline defaults/main.yml]: please add zookeeper_ prefix

File: README.md [hunk 2]
```diff
@@ -53,6 +53,7 @@ tick_time: 2000
 zookeeper_autopurge_purgeInterval: 0
 zookeeper_autopurge_snapRetainCount: 10
 zookeeper_cluster_ports: "2888:3888"
+max_client_connections: 60
```
Comment (ernestas-poskus) [inline README.md]: please add zookeeper_ prefix

File: templates/zoo.cfg.j2 [hunk 3]
```diff
@@ -4,7 +4,7 @@ dataLogDir={{ log_dir }}
 clientPort={{ client_port }}
 initLimit={{ init_limit }}
 syncLimit={{ sync_limit }}
-
+maxClientCnxns={{ max_client_connections }}
```
Comment (ernestas-poskus) [inline templates/zoo.cfg.j2]: please add zookeeper_ prefix

**Comment by emalloy** [comment]
@ernestas-poskus Done! Thanks for your quick turnaround!

**Pull request merged**